app/services/ember-cli.js

```diff
@@ -1,7 +1,22 @@
 import Ember from "ember";
 import Babel from "npm:babel";
+import Path from 'npm:path';
 import blueprints from '../lib/blueprints';
-import config from "../config/environment";
+
+const twiddleAppName = 'demo-app';
+
+// These files will be included if not present
+const boilerPlateJs = [
+  'app',
+  'config/environment',
+  'router',
+  'initializers/router'
```

Comment:
it's not, I just want the code to live in it's own file instead of being embedded inline.